src/docs/reference/core/Apis.GraphQL.Abstractions/README.md

```diff
@@ -178,9 +184,15 @@ Shown in the example above, we have an autoroutePart argument, this is registere
 
 Done.
 
-### Extending Content Type Query Filters
+### Using default Content Type query filters
+
+In the previous section, we demonstrated how to create filters for complex requirements, allowing you to create custom filtration methods. However, in case you need to add filter on Content Type queries, there is also a simpler solution for many usecases.
```

Comment:
```suggestion
### Using default Content Type query filters

In the previous section, we demonstrated how to create filters for complex requirements, allowing you to create custom filtration methods. However, in case you need to add a filter on Content Type queries, there is also a simpler solution for many use cases.
```